source/Generic/LibraryExporter/ViewModels/LibraryExporterViewModel.cs

```diff
@@ -244,6 +245,12 @@ private string[] GenerateGameColumns(int columnCount, Game game)
                 currentInsertColumn++;
             }
 
+            if (Settings.Settings.ExportSettings.CompletionStatus)
+            {
+                properties[currentInsertColumn] = !string.IsNullOrEmpty(game.CompletionStatus.Name) ? game.CompletionStatus.Name : string.Empty;
```

Comment:
I think here it should be checked if `CompletionStatus` is null. Also for consistency use the already available `IsNullOrEmpty` extension method instead, like `!game.CompletionStatus.Name.IsNullOrEmpty()`

Reply:
Ok, I guess it's ok like that:
`properties[currentInsertColumn] = (game.CompletionStatus != null && !game.CompletionStatus.Name.IsNullOrEmpty() ? game.CompletionStatus.Name : string.Empty);`